### PR TITLE
Update files to remove imhameed

### PR DESCRIPTION
**Pull request by SamMonoRT**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -34,10 +34,10 @@
...
-/src/mono/mono/mini @vargaz @lambdageek @SamMonoRT @imhameed
+/src/mono/mono/mini @vargaz @lambdageek @SamMonoRT
 /src/mono/mono/mini/*cfgdump* @vargaz
-/src/mono/mono/mini/*exceptions* @vargaz @BrzVlad @imhameed
-/src/mono/mono/mini/*llvm* @vargaz @imhameed @EgorBo
+/src/mono/mono/mini/*exceptions* @vargaz @BrzVlad
+/src/mono/mono/mini/*llvm* @vargaz @EgorBo
 /src/mono/mono/mini/*ppc* @vargaz
 /src/mono/mono/mini/*profiler* @BrzVlad @lambdageek
 /src/mono/mono/mini/*riscv* @alexrp
```
Comment (jeffhandley) [inline .github/CODEOWNERS:43]: Can you confirm that this line is still correct?

**Review by jeffhandley — APPROVED** [review]
Pending one unrelated question, looks good. I did a search to confirm this covered all the places that needed to be updated.

File: .github/CODEOWNERS (hunk 1 above)
Comment (jeffhandley) [reply .github/CODEOWNERS:43]: Specifically, alexrp shows with an error that they don't have permissions.

Comment (SamMonoRT) [reply .github/CODEOWNERS:43]: seems right, Egor moved to Juliee's team but has been helping with reviews as necessary. We can leave it that way

Comment (SamMonoRT) [reply .github/CODEOWNERS]: > Specifically, alexrp shows with an error that they don't have permissions.

updated that, thanks for catching that.

**Comment by EgorBo** [comment]
Could you please also remove me from mono related paths? Thanks

**Comment by SamMonoRT** [comment]
> Could you please also remove me from mono related paths? Thanks

done

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -53,7 +53,7 @@
 /src/mono/mono/utils/atomic* @vargaz
 /src/mono/mono/utils/mono-hwcap* @vargaz
 /src/mono/mono/utils/mono-mem* @vargaz
-/src/mono/mono/utils/mono-merp* @lambdageek @naricc @imhameed
+/src/mono/mono/utils/mono-merp* @lambdageek @naricc
 /src/mono/mono/utils/mono-state* @lambdageek
```
Comment (lambdageek) [inline .github/CODEOWNERS:57]:
```suggestion
```

MERP (and mono-state which it used) are gone.  Ok to just delete this now.

**Pull request merged**